src/main.cpp

```diff
@@ -11,139 +11,217 @@
 #include "state.hpp"
 #include "camera.hpp"
 
+GLuint loadTexture(const char* texImagePath) {
+	GLuint textureID;
+	textureID = SOIL_load_OGL_texture(texImagePath, SOIL_LOAD_AUTO, SOIL_CREATE_NEW_ID, SOIL_FLAG_INVERT_Y);
+	if (textureID == 0)
+		printf("could not find texture file\n");
+	return textureID;
+}
+
```

Comment:
asta ar merge integrata in contructoru de la planeta, ca gen ala primeste un filepath pentru textura